versions/1/integration/manifest.spec.yml

```diff
@@ -1,5 +1,5 @@
 ##
-## Describes the specification for the package's main manifest.yml file
+## Describes the specification for the integration's main manifest.yml file
```

Comment:
See below, I think we should talk about integration packages.

Reply:
Fixed